src/measure_theory/measure_space.lean

```diff
@@ -2258,6 +2258,152 @@ lemma measurable.comp_ae_measurable [measurable_space δ] {f : α → δ} {g : 
 
 end
 
+section ae_seq
```

Comment:
I think this could go to its own file: the file `measure_space.lean` is already too big, and this is self-contained and not used elsewhere in the file.